tests/functional/base_pages/introduction.page.js

```diff
@@ -5,8 +5,8 @@ class IntroductionPage extends BasePage {
     return "#my-account";
   }
 
-  signOut() {
-    return '[data-qa="btn-sign-out"]';
+  exitButton() {
```

Comment:
Not sure about this, are you not still signed in on the introduction?

Reply:
This was requested by Joe